### ISSUE TITLE
Version 3.6.12 option ADD embedded title doesn't work on my computer

### ISSUE THREAD

**Issue by Cyberyoda1411** [SubtitleEdit/subtitleedit#6777]
Version 3.6.12. Option Video/"Generate video with added/removed embedded subtitles" doesn't work. That is, REMOVE option works, but I can't ADD embedded subtitles, because my system 'eats' more than half of the original subtitle (srt file). For example, my original srt file has 229 lines and the video with embedded file has only 49 lines. Here is the proof. This is the screenshot of the embedded video, made by Subtitle edit, only 49 lines of subtitle, instead of 229:
 
![Embedded srt](https://user-images.githubusercontent.com/128191515/227807027-cee78d1a-4854-4222-bc2d-4556834f8f08.jpg)

This is the original subtitle, with 229 lines:

![Originalni srt](https://user-images.githubusercontent.com/128191515/227807546-dfd404b8-f1f6-434b-8f01-1cc8878c81c9.jpg)

More information:
1. I have FFmpeg 6.0, I put its bin folder in environment path, and the program itself adds folder with the newest FFmpeg. So, I don't think this is the problem, the program can reach FFmpeg from two sources. 
2. I tested it with the shorter path, I thought the path where video is has too many characters, it doesn't work
3. I tried to add/remove embedded subtitles with the program MKVToolNix and it works perfectly. 
4. I tested this with the bigger files, about 700 MB, and it is the same thing. Program Subtitle Edit made the video with embedded subtitles, but it 'ATE' more than half of lines. I think, the original subtitle had over 860 lines, and I got 400 in the embedded subtitle video. 
5. I use Windows 10.

WHAT IS THE PROBLEM? Any ideas? How MKVToolNix works perfectly, but this option in SE doesn't work as it should? I know that author of this program tested this features and they work on his computer. What am I missing?

**Comment by niksedk** [comment]
Could you try to this ffmpeg 6: https://www.gyan.dev/ffmpeg/builds/ffmpeg-release-essentials.zip ?

You can try running the SE generated ffmpeg command in a command line... get the parameters in SE by right-clicking on the "Generate" button, and run with prompt for parameters:
![image](https://user-images.githubusercontent.com/1261331/227848231-4d730168-e0eb-400f-8ff2-3eced7692cf3.png)

**Comment by niksedk** [comment]
Could you try:
1) Uninstall all of SE + delete any leftover SE folders.
2) Re-install SE + download ffmpeg via SE (it's now ffmpeg 6)
3) Try to add an embedded srt file again

**Comment by Cyberyoda1411** [comment]
I continue my research. I removed K-lite codec pack, but it didn't change anything. 

I understand why you are focused on FFmpeg, because the operation of embedded coding is performed by the command line of this component. So, the problem should be solved if we realize what is wrong with FFmpeg.

**Comment by Cyberyoda1411** [comment]
I tried it on several subtitles, but OK, here is the attachment of srt. 
[S01E01 .zip](https://github.com/SubtitleEdit/subtitleedit/files/11080221/S01E01.zip)

**Comment by niksedk** [comment]
Thx for the sub file :)
How is this beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.12/SubtitleEditBeta.zip ?

**Comment by Cyberyoda1411** [comment]
It works! I got 229 lines of subtitles in the embedded subtitle. I tested it on the mp4 file, and the output files were mp4 and mkv, it is the same, and it works fine, 229, lines, your program doesn't eat any subtitle lines. 

Now, EXPLAIN to me what was the problem. And will you make some corrections to the latest release, or I should use this beta? 

You had some message about ANSI encoding. I can see that your OUTPUT format of the embedded subtitles is **UTF-8 with BOM**, while my original srt is **1250 Central Europe**. I guess, this was the problem?
